install/kubernetes/cilium/values.yaml

```diff
@@ -69,6 +69,18 @@ affinity:
           - cilium
       topologyKey: kubernetes.io/hostname
 
+affinityOperator:
```

Comment:
should this one be inside operator object insteadly ?

Reply:
CI failure seems related 

```
Error: unable to build kubernetes objects from release manifest: error validating "": error validating data: ValidationError(Deployment.spec.template.spec.affinity): unknown field "requiredDuringSchedulingIgnoredDuringExecution" in io.k8s.api.core.v1.Affinity
```

Reply:
Indeed. My point was to have consistency with what was done for the agent daemonset. 
I'm gonna give it a try and keep you posted.

Reply:
@sayboras the CI failure happened because the `indent` was not indenting properly the affinity block. I swapped to nindent and this solved the issue.
Now I've incorporated your suggestion and I moved the affinity inside the operator object.